catkin_ws/src/image_rotation/src/image_rotation_publisher.cpp

```diff
@@ -0,0 +1,47 @@
+///
+/// @file
+/// @brief Implementation of ROS Publisher
+///
+
+#include <iostream>
+
+#include "../include/image_rotation/image_algorithm.h"
+
+#include <cv_bridge/cv_bridge.h>
+#include <opencv2/imgproc/imgproc.hpp>
```

Comment:
Check the need imgproc and highgui header files

Reply:
extra header cv bridge . not required here .

Reply:
done